backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVActionFlowTests.java

```diff
@@ -274,6 +274,38 @@ public void reconnectTransitionUsageTarget() {
         });
     }
 
+    @Sql(scripts = { ActionFlowCompartmentTestProjectData.SCRIPT_PATH }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
Please add ", config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))" on each @Sql annotation
This is missing in many of our tests, but needed.